docs/prometheus-exporter.md

```diff
@@ -150,3 +150,33 @@ Always has the value 1.
 
 Timestamp of starting up the node (Unix time in milliseconds).
 
+### `grpc_request_response_time_seconds`
+
+Histogram tracking the total number of gRPC requests received and the time it took to provide a response. Labelled with the gRPC method name (`method=<name>`) and the gRPC response status (`status=<status>`).
+
+The duration tracked is time it takes for the underlaying HTTP request to construct a response, which does not include the time it takes to stream the response to the client.
```

Comment:
"The duration tracked is time it takes for the underlaying HTTP request to construct a response"

Does not really make sense. Do you mean that the time it takes for the handler to construct a response? I don't think we need to be mentioning HTTP here at all.

Reply:
I have rephrased this now